app/helpers/orchid/date_helper.rb

```diff
@@ -0,0 +1,75 @@
+module Orchid::DateHelper
+
+  def clear_dates_params
+    # duplicate parameters, remove date fields, and return
+    options = params.permit!.deep_dup
+    options.delete("date_from")
+    options.delete("date_to")
+    return options
+  end
+
+  # TODO could abstract this into a CDRH gem
+  def date_default(date_params, default_date)
+    y, m, d = date_params.map { |x| x.gsub(/\s+/, "") }
```

Comment:
I honestly don't remember why this got added this way, it's not clear in the original commit from L&C:
https://github.com/CDRH/lewisandclark/commit/ca961ab15cd9c708793a4333599a3570a424e0c9

I'm happy to change it to strip and then if any funny business comes up we can fix it.  Should I apply the fix to L&C, also?